egs/tedlium/s5/local/chain/run_tdnn.sh

```diff
@@ -1,7 +1,22 @@
 #!/bin/bash
 #
 # This script requires that you have run the toplevel run.sh script in TEDLIUM up to stage 7.
-#
+# This script is designed to run using the 'cleaned-up' data.
+
```

Comment:
Remove this blank line, it will make the comments read better (and make the line above less confusing).